src/api/app/assets/stylesheets/webui2/modals.scss

```diff
@@ -5,3 +5,12 @@
 .modal-body p {
     white-space: pre-wrap;
 }
+
+.modal-footer {
+  a {
```

Comment:
Okay, then shouldn't this be generic? Why is this `modal-footer` specific?

Reply:
You are right!